src/components/ebay-carousel/utils/scroll-transition/index.js

```diff
@@ -73,6 +78,7 @@ module.exports = function scrollTransition(el, to, fn) {
  * @param {number} val - A number between 0 and 1.
  * @return {number}
  */
-function easeInOut(v) {
-    return v < 0.5 ? 2 * v * v : -1 + (4 - 2 * v) * v;
+function easeOut(v) {
+    const t = v - 1;
+    return t * t * t + 1;
```

Comment:
I know it's merged but `Math.pow()`, FTW.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/pow